packages/rum/test/viewCollection.spec.ts

```diff
@@ -116,23 +119,32 @@ describe('rum track renew session', () => {
 })
 
 describe('rum track load duration', () => {
-  let lifeCycle: LifeCycle
+  let setupBuilder: TestSetupBuilder
+  let addRumEvent: jasmine.Spy
   let getViewEvent: (index: number) => View
 
   beforeEach(() => {
-    jasmine.clock().install()
-    ;({ lifeCycle, getViewEvent } = spyOnViews())
+    ;({ addRumEvent, getViewEvent } = spyOnViews())
+
+    const fakeLocation: Partial<Location> = { pathname: '/foo' }
+    mockHistory(fakeLocation)
```

Comment:
Maybe `mockHistory` could be part of the setup builder?

Reply:
I think we can still have helper specific to spec files when it is not used elsewhere.
I'd be in favor of moving it there if we need to reuse it in another context.

Reply:
Ok!